src/crawlers/cheerio_crawler.js

```diff
@@ -464,71 +473,68 @@ class CheerioCrawler {
     /**
      * Wrapper around handlePageFunction that opens and closes pages etc.
      *
-     * @param {Object} options
-     * @param {Request} options.request
-     * @param {AutoscaledPool} options.autoscaledPool
-     * @param {Session} [options.session]
+     * @param {Object} crawlingContext
+     * @param {Request} crawlingContext.request
+     * @param {AutoscaledPool} crawlingContext.autoscaledPool
+     * @param {Session} [crawlingContext.session]
      * @ignore
      */
-    async _handleRequestFunction({ request, autoscaledPool, session }) {
-        let proxyInfo;
+    async _handleRequestFunction(crawlingContext) {
         let proxyUrl;
+
```

Comment:
Same here. Once you use the `crawlingContext` as the parameter in the function, you can freely destructure the objects. It will save you having to type `crawlingContext....` everywhere.

```js
async _handleRequestFunction(crawlingContext) {
    const { request, session, proxyInfo } = crawlingContext;
    // ...
```